### PR TITLE
[Observability] wrangler observability binding configuration

### PR DESCRIPTION
**Pull request by rohinlohe**
- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by irvinebroque — COMMENTED** [review]
Yes, needs to be here but there is a lot more to say and should be possible to grok in the docs outside of very very long wrangler configuration docs (that are overloaded and need a rethink)

e.g. this section? https://developers.cloudflare.com/workers/observability/

File: src/content/docs/workers/wrangler/configuration.mdx [hunk 1]
```diff
@@ -678,6 +681,33 @@ binding = "<BINDING_NAME2>"
 id = "<NAMESPACE_ID2>"
 ```
 
+### Observability
+
+The [Observability](/workers/observability) binding provides your Worker with automatic persistent logs that can be searched, filtered, and queried directly from the Workers dashboard.
```
Comment (zebp) [inline src/content/docs/workers/wrangler/configuration.mdx]: *techinically* this isn't a binding since it isn't accessible to the user in the env object

**Comment by rohinlohe** [comment]
Agree, @irvinebroque. This is just documenting the wrangler changes to unblock the wrangler PR.

**Review by db-cloudflare — APPROVED** [review]

**Review by irvinebroque — APPROVED** [review]
lfg

**Pull request merged**